### PR TITLE
VACMS-19649: Update Connect with us queries & templates for CLPs and BHLPs

### PR DESCRIPTION
**Pull request by davidmpickett**
## Summary

- BHLPs and CLPs queries and templates now point to a new Connect With Us Content Block instead of the previous Office Content Type.
- All fields on Connect With Us Content Block are named the same as those on Office Content Type
- @department-of-veterans-affairs/vfs-public-websites-frontend  

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19544
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19649

## Testing done

[Manual test plan](https://docs.google.com/spreadsheets/d/1wtRSKT0MzitaPRaEvLkhjBro5ma__n48vahGWV5x-ws/edit?gid=0#gid=0)

## Screenshots

Product | Before | After 
-- | -- | -- 
BHLP Mobile  | Screenshot 2024-11-20 at 2 00 38 PM | Screenshot 2024-11-20 at 1 57 54 PM
BHLP Desktop | Screenshot 2024-11-20 at 1 39 46 PM | Screenshot 2024-11-20 at 2 01 08 PM
CLP Mobile  | Screenshot 2024-11-20 at 2 00 56 PM | Screenshot 2024-11-20 at 1 58 14 PM
CLP Desktop | Screenshot 2024-11-20 at 1 49 32 PM | Screenshot 2024-11-20 at 1 58 03 PM

## What areas of the site does it impact?

**BHLPs**
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/family-member-benefits
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/service-member-benefits
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/housing-assistance
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/life-insurance
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/family-and-caregiver-benefits
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/disability
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/pension
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/careers-employment

**CLPs**
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/initiatives/protecting-veterans-from-fraud
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/initiatives/va-resources-for-survivors-of-suicide-loss
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/initiatives/veteran-trust-in-va
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/initiatives/covid-flu

**Nodes in revision Limbo** (Fixed in Prod, but not in Tugboat)
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/health-care
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/burials-memorials
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/decision-reviews
- https://web-sivbehcvqkui7ce9b67dwanfbohmenin.demo.cms.va.gov/initiatives/prepare-for-vas-secure-sign-in-changes

## Acceptance criteria
- [ ] On a BHLP or CLP, if a Connect With Us Block is added in Drupal, it displays on FE
- [ ] On a BHLP or CLP, if a Connect With Us Block is not added in Drupal, nothing displays on FE
- [ ] Only Published Revisions of Connect with Us Blocks display on FE, Draft revisions do not

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/registry.json [hunk 1]
```diff
@@ -506,32 +507,6 @@
       ]
     }
   },
-  {
```
Comment (jilladams) [inline src/applications/registry.json:509]: Looks like some code changes in this file got included in your PR maybe from a rebase. Flagging for review.

File: src/site/includes/header.html [hunk 2]
```diff
@@ -72,13 +72,7 @@
   <link rel="preload" href="/generated/fa-solid-900.woff2" as="font" type="font/woff2" crossorigin>
 
   <!-- CSS -->
-  {% comment %}
```
Comment (jilladams) [inline src/site/includes/header.html:75]: Same here - guessing this got introduced via rebase, needs revew.

File: src/applications/registry-schema.json [hunk 3]
```diff
@@ -20,6 +20,9 @@
             "rootUrl": {
                 "type": "string"
             },
+            "useLocalStylesAndComponents": {
+                "type": "boolean"
+            },
```
Comment (jilladams) [inline src/applications/registry-schema.json:25]: not sure, but seems like this may also be an accidental inclusion via rebase.

**Review by jilladams — DISMISSED** [review]
Seems like some code changes included here don't pertain to this ticket, and may need review or help to get them excluded again or pull whatever's true in main into this branch - just noting that in the places I saw it while I was taking a look. (I'm not making this a "Request changes" review, so yo won't be blocked on my approval.)

**Review by chriskim2311 — APPROVED** [review]
LGTM! Tested in RI and code changes look complete.

**Pull request merged**